third_party/docfx/templates/devsite/layout/_master.tmpl

```diff
@@ -5,51 +5,28 @@
 {{!include(logo.svg)}}
 {{!include(search-stopwords.json)}}
 <!DOCTYPE html>
-<!--[if IE]><![endif]-->
 <html devsite>
   {{>partials/head}}
-  <body data-spy="scroll" data-target="#affix" data-offset="120">
-    <div id="wrapper">
+  <body>
       <header>
-        {{^_disableNavbar}}
-        {{>partials/navbar}}
-        {{/_disableNavbar}}
-        {{^_disableBreadcrumb}}
-        {{>partials/breadcrumb}}
-        {{/_disableBreadcrumb}}
       </header>
       {{#_enableSearch}}
       <div class="container body-content">
         {{>partials/searchResults}}
       </div>
       {{/_enableSearch}}
       <div role="main" class="container body-content hide-when-search">

```

Comment:
Looks like this can be simplified, too.

Reply:
trimmed attributes except `data-uid` on the `article`